hardware/Pipfile

```
@@ -0,0 +1,24 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+python-can = "==3.3.4"
+pyserial = "==3.5"
+typing-extensions = "==3.10.0.0"
+
+[dev-packages]
+pytest = "==6.1.0"
+pytest-aiohttp = "==0.3.0"
+pytest-cov = "==2.10.1"
+mypy = "==0.800"
```

Oh oops, can we use latest mypy to match `api` and `robot-server`? I think it's `0.910`

```suggestion
mypy = "==0.910"
```